common/tsconfig/tsconfig.json

```diff
@@ -1,3 +1,102 @@
 {
-  "extends": "namseent/tsconfig.json"
+  "compilerOptions": {
```

Comment:
I tried to make it `../../tsconfig.json` but it make a circular dependency because directory becomes in node_modules.